projects/stargazer/plugins/other/radius/server.cpp

```diff
@@ -42,27 +55,33 @@ RadProto::Packet Server::makeResponse(const RadProto::Packet& request)
 
 void Server::handleSend(const error_code& ec)
 {
-    if (ec)
-        std::cout << "Error asyncSend: " << ec.message() << "\n";
+    if (m_token.stop_requested())
+        return;
 
+    if (ec)
+    {
+        m_logger("Error asyncSend: %s", ec.message());
+        printfd(__FILE__, "Error asyncSend: '%s'\n", ec.message());
```

Comment:
`ec.message()` returns `std::string`, while '%s' requires `const char*`. Please add `.c_str()` here and in other places.